src/components/feedback/SubmittedComponent.tsx

```diff
@@ -39,7 +41,7 @@ const SubmittedComponent = ({
   };
 
   return (
-    <View style={styles.container}>
+    <View style={styles.container} ref={focusRef}>
```

Comment:
Testet på Android? Jeg tror man må ha `accessible={true}` for at dette skal fungere på Android. 

Tror det må endres litt på strukturen her. Man kan ikke sette `accessible` direkte på denne `View`-komponenten, siden det gjør at man ikke klarer å navigere inn på underliggende button med skjermleser.

Reply:
accessible er vel true by default

Reply:
Ikke helt. Elementet er accessible også ved å ikke sette flagget, men om man setter `accessible={true}` så sier man at dette nivået er det skjermleseren skal aksessere. Så hvis man har en `View`  med tre underliggende `ThemeText` så vil man ved `accessible={true}` kun ha mulighet for å sette fokus på `View`-et, mens uten `accessible` så vil skjermleseren kunne hoppe innom hvert underliggende `ThemeText`.